### PR TITLE
#2872 [Manager] Fix stream app delete and start issue

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2872)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/pages/PipelinePage/Connectors/StreamApp.js [hunk 1]
```diff
@@ -226,7 +226,18 @@ const StreamApp = props => {
 
     const connectorHasDeleted = get(connectorResponse, 'data.isSuccess', false);
 
-    const updatedFlows = flows.filter(flow => flow.from.name !== streamAppName);
+    const updatedFlows = flows
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/PipelinePage/Connectors/StreamApp.js]: Make sure stream app `from` and `to` can both be cleared when removing a stream app from the graph

File: ohara-manager/client/src/components/pages/PipelinePage/PipelineEditPage/Operate.js [hunk 2]
```diff
@@ -51,15 +49,15 @@ const Operate = props => {
       );
 
       streamsPromises = streams.map(stream =>
-        streamApi.startStreamApp(streamGroup, stream),
+        streamApi.startStreamApp(pipelineGroup, stream),
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/PipelinePage/PipelineEditPage/Operate.js]: Wrong group name...

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by chia7712** [comment]
retry manager-e2e

**Pull request merged**